modules/renderer/src/text-measurer.ts

```diff
@@ -34,17 +34,19 @@ export default class TextMeasurer {
     this.ctx.font = `${fontStyle} ${fontWeight} ${fontSize}px ${fontFamily}`
   }
 
-  measure(text: string): Size {
-    const {fontSize, lineHeight} = this.opts
-    const rowHeight = fontSize * 1.2
-    const rowSpacing = fontSize * (lineHeight - 1)
+  measure(text: string, fontSize: number, fontFamily: string, fontStyle = 'normal'): Size {
     let w = 0
+    let h = 0
     const lines = text.split('\n')
+    this.ctx.font = fontStyle + ' ' + fontSize.toString() + ' ' + fontFamily
+    let lineH: number
     for (const line of lines) {
       const metrics = this.ctx.measureText(line)
       w = Math.max(w, metrics.width)
+      lineH = metrics.actualBoundingBoxAscent
```

Comment:
This is not supported in any browser other than Chrome. https://caniuse.com/?search=textmetrics

Reply:
According to https://developer.mozilla.org/en-US/docs/Web/API/TextMetrics/actualBoundingBoxAscent, only InternedExplorer does not support it.